Formula/bazel.rb

```diff
@@ -15,6 +16,9 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Bazel has it owns sandbox, ensure it works.
+    ENV.delete("SDKROOT")
+    ENV["CC"] = `xcrun -f #{ENV["CC"]}`.strip
```

Comment:
the superenv wrapper use the environment. Bazel does strip out the environment when building. So we get the "requires --std=env" if we use the wrapper from the superenv. This is just replaying the way superenv find the C++ compiler.